docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -111,87 +149,246 @@ It would help getting a bit more organized if our restaurants directory had some
 8. Choose the Relation field.
 9. In the center, select the icon that represents "many-to-many" ![icon many-to-many](/img/assets/quick-start-guide/icon_manytomany.png). The text should read `Categories has and belongs to many Restaurants`.
 
-![Admin Panel screenshot: relations](/img/assets/quick-start-guide/qsg-handson-part2-02-collection_ct.png)
+<ThemedImage
+alt="Admin Panel screenshot: relations"
+sources={{
+  light: '/img/assets/quick-start-guide/qsg-handson-part2-02-collection_ct.png',
+  dark: '/img/assets/quick-start-guide/qsg-handson-part2-02-collection_ct_DARK.png',
+}}
+/>
 
 11. Finally, click **Finish**, then the **Save** button, and wait for Strapi to restart.
 
-### Step 2: Use the collection types to create new entries
+</details>
+
+:::callout 🥳 CONGRATULATIONS!
+You have just created a basic data structure for your Strapi project! You can keep on playing with the [Content-type Builder](/user-docs/content-type-builder), or proceed to parts C and D below to discover Strapi Cloud and add actual content to your project.
+:::
+
+## ☁️ Part C: Deploy to Strapi Cloud
+
+Now that your beautiful first Strapi project is working locally, it's time for the world to see it live! The most straightforward way to host your project is to use Strapi Cloud. Before deploying to Strapi Cloud, you will need to host your Strapi project on an online repository — we will use GitHub.
+
+<details style={detailsStyle}>
+<summary style={summaryStyle}>Step 1: Host the code of your Strapi project on GitHub</summary>
 
-Now that we have created a basic data structure with 2 collection types, "Restaurant" and "Category", let's use them to actually add content by creating new entries.
+### Step 1: Host the code of your Strapi project on GitHub
 
-#### Create an entry for the "Restaurant" collection type
+Create a new GitHub repository and push the code of your Strapi project to this repository. If you're not already familiar with GitHub, the togglable content below should get you started 👇
 
-1. Go to ![Content Manager icon](/img/assets/quick-start-guide/icons/content.svg) [Content Manager > Collection types - Restaurant](http://localhost:1337/admin/content-manager/collectionType/api::restaurant.restaurant) in the navigation.
+<details>
+<summary>Click me to view the steps required to push your Strapi project code to GitHub:</summary>
```

Comment:
This is a very subjective comment but I'm not super convinced by the "Click me" 😂  I think you already indicate above that people should follow the toggled content, so the title of the toggle could just be something like "Steps required to push..." like you did with the main, numbered steps. Obviously, very subjective so I let you decide what to keep in the end 😄

Reply:
Yes, I wasn't sure when I wrote it, but that's a good idea :)